crates/ethportal-api/src/types/content_key/history.rs

```diff
@@ -32,8 +32,16 @@ pub enum HistoryContentKey {
     BlockBody(BlockBodyKey),
     /// The transaction receipts for a block.
     BlockReceipts(BlockReceiptsKey),
-    EphemeralHeadersByFindContent(EphemeralHeadersByFindContentKey),
-    EphemeralHeaderByOffer(EphemeralHeaderByOfferKey),
+    /// Ephemeral headers by find content.
+    ///
+    /// This is used to find the headers of a block and its ancestors.
+    /// This response can only be used by FindContent requests.
+    EphemeralHeadersFindContent(EphemeralHeadersFindContentKey),
+    /// Ephemeral header offer.
+    ///
+    /// This is used to offer a block header to a peer. This response can only be used by Offer
+    /// requests. This type contains a single header.
```

Comment:
```suggestion
    /// Ephemeral header used for Offer request..
    ///
    /// This is used to offer an ephemeral block header to a peer. This content type can only be used by Offer
    /// requests. The type contains a single header, but more headers should be included in the single Offer request.
```

I would add more info. Feel free to rephrase it \ formulate it better.